### PR TITLE
BUGFIX: Auth + Profile

### PR DESCRIPTION
**Pull request by MaximeZmt**
Correction of google signin bug (recreating a new user each login) + MyProfile change profile picture text + add default profile picture in profile activity

### CODE REVIEW

File: app/src/main/java/ch/sdp/vibester/activity/AuthenticationActivity.kt [hunk 1]
```diff
@@ -158,7 +158,9 @@ class AuthenticationActivity : AppCompatActivity() {
                     Log.d(getString(R.string.log_tag), "signInWithCredential:success")
                     if (task.getResult().additionalUserInfo != null) {
                         createAcc = task.getResult().additionalUserInfo!!.isNewUser
-                        createAccount()
+                        if (createAcc){
```
Comment (jiabaow) [inline app/src/main/java/ch/sdp/vibester/activity/AuthenticationActivity.kt:161]:
```suggestion
                        if (createAcc) {
```

File: app/src/main/java/ch/sdp/vibester/activity/profile/MyProfileActivity.kt [hunk 2]
```diff
@@ -66,7 +66,7 @@ class MyProfileActivity : ProfileActivity() {
      * helper function to show dialog 
      */
     private fun showDialogWhenChangeImage() {
-        showGeneralDialog(R.string.profile_verify_change_profile_pic.toString(), false)
+        showGeneralDialog(getString(R.string.profile_verify_change_profile_pic), false)
```
Comment (jiabaow) [inline app/src/main/java/ch/sdp/vibester/activity/profile/MyProfileActivity.kt:69]: what's the difference between getString and toString?

**Review by jiabaow — APPROVED** [review]
LGTM

**Review by kamilababayeva — APPROVED** [review]
LGTM!!!!

**Review by zwierski — APPROVED** [review]
LGTM 👍

**Pull request merged**